source/github-events/on-new-comments.ts

```diff
@@ -50,20 +50,18 @@ function addListeners(): delegate.Subscription[] {
 		childList: true,
 	});
 
-	return [
-		// When hidden comments are loaded by clicking "Load more…"
-		delegate(document, '.js-ajax-pagination', 'submit', paginationSubmitHandler),
+	// When hidden comments are loaded by clicking "Load more…"
+	delegate(document, '.js-ajax-pagination', 'submit', paginationSubmitHandler, {signal});
 
-		// Collapsed comments are loaded later using an include-fragment element
-		delegate(document, 'details.js-comment-container include-fragment:not([class])', 'loadstart', getFragmentLoadHandler(run), true),
-	];
+	// Collapsed comments are loaded later using an include-fragment element
+	delegate(document, 'details.js-comment-container include-fragment:not([class])', 'loadstart', getFragmentLoadHandler(run), {capture: true, signal});
 }
 
-export default function onNewComments(callback: VoidFunction): Deinit {
+export default function onNewComments(callback: VoidFunction, signal: AbortSignal): Deinit {
+	addListeners(signal);
 	handlers.add(callback);
 
 	return [
-		...addListeners(),
 		handlers.clear,
```

Comment:
I don’t think this ever worked? `Set#clear` isn't bound

<img width="364" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/175786147-5bd1d17c-8492-46e2-ae0e-f512ba983291.png">

Reply:
Fixed in ff282b95